### PR TITLE
Adding OS metrics role

### PR DESCRIPTION
**Pull request by nivancevic**

### CODE REVIEW

File: roles/osmetrics/files/osmetrics.service [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=osmetrics
+After=network.target
+
+[Service]
+ExecStart=/usr/share/osmetrics/osmetrics
```
Comment (mgobec) [inline roles/osmetrics/files/osmetrics.service]: Path should be based on install dir variable

File: roles/osmetrics/files/osmetrics [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+[ -f /etc/osmetrics/config ] && . /etc/osmetrics/config
```
Comment (mgobec) [inline roles/osmetrics/files/osmetrics]: Path should be based on configuration dir variable

File: roles/osmetrics/files/osmetrics [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+/usr/bin/python /usr/share/osmetrics/start.py -h $OSMETRICS_RIEMANN_HOST -p $OSMETRICS_RIEMANN_PORT -i $OSMETRICS_INTERVAL
```
Comment (mgobec) [inline roles/osmetrics/files/osmetrics]: Path should be based on install dir variable

**Comment by nbozic-zz** [comment]
Add in commit message it is closing #6 or close that issue when finished

**Pull request merged**